src/main/java/net/atos/zac/app/zaken/converter/RESTZaaktypeConverter.java

```diff
@@ -37,7 +40,8 @@ public RESTZaaktype convert(final Zaaktype zaaktype) {
         restZaaktype.opschortingMogelijk = zaaktype.getOpschortingEnAanhoudingMogelijk();
         restZaaktype.verlengingMogelijk = zaaktype.getVerlengingMogelijk();
         if (restZaaktype.verlengingMogelijk) {
-            restZaaktype.verlengingstermijn = zaaktype.getVerlengingstermijn().getDays();
+            LocalDateTime start = LocalDateTime.now();
+            restZaaktype.verlengingstermijn = Long.valueOf(start.until(start.plus(zaaktype.getVerlengingstermijn()), ChronoUnit.DAYS)).intValue();
```

Comment:
Verplaats de logica naar een method in de PeriodUtil class. Zo kunnen we het ook op andere plekken gebruiken.
Maak een method: int aantalDagenVanafHeden(final Period period)

Reply:
Verplaatst in opvolgende commit